ui/de-lib/src/main/java/org/iplantc/de/admin/desktop/client/metadata/view/TemplatesListingViewImpl.java

```diff
@@ -123,6 +150,7 @@ public String getKey(MetadataTemplateInfo item) {
                                                                                     appearance.deleted());
         List<ColumnConfig<MetadataTemplateInfo, ?>> columns = new ArrayList<ColumnConfig<MetadataTemplateInfo, ?>>();
         columns.add(nameCol);
+        columns.add(descriptionCol);
```

Comment:
Sure, I can remove the column.  It looks like if you do have a lengthy description it gets truncated in the column view after however many characters, at least.

Either way, this made me realize I should use `TextArea` in the edit window for description instead of `TextField` so I'll update that real quick.